package.json

```diff
@@ -145,5 +146,11 @@
     "vscode-editorconfig": "https://open-vsx.org/api/EditorConfig/EditorConfig/0.14.4/file/EditorConfig.EditorConfig-0.14.4.vsix",
     "vscode-eslint": "https://open-vsx.org/api/dbaeumer/vscode-eslint/2.1.1/file/dbaeumer.vscode-eslint-2.1.1.vsix",
     "vscode-references-view": "https://open-vsx.org/api/ms-vscode/references-view/0.0.47/file/ms-vscode.references-view-0.0.47.vsix"
+  },
+  "dependencies": {
+    "@theia/node-pty": "0.9.0-theia.6",
+    "nsfw": "1.2.9",
+    "native-keymap": "2.1.2",
+    "find-git-repositories": "0.1.2"
```

Comment:
This must be the side-effect of a failed `yarn rebuild:browser && yarn rebuild:electron` execution. This does not belong here!